test/sorting.jl

```diff
@@ -842,5 +841,6 @@ end
         end
     end
 end
+# The "searchsorted" testset is at the end of the file because it is slow.
```

Comment:
I added these comments because by default I put new tests at the end of the file and these comments remind me not to. (see #45233, #45234)